apine.schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#", 
+  "$id": "https://github.com/INCF/apine/blob/master/apine.schema.json", 
+  "type": "object", 
+  "additionalProperties": false, 
+  "properties": {
+    "dataset": {
+      "$id": "https://github.com/INCF/apine/blob/master/apine.schema.json/dataset", 
+      "minLength": 1,
+      "type": "string", 
+      "title": "dataset", 
+      "description": "The parent" 
+    },
+    "participant": {
+      "$id": "https://github.com/INCF/apine/blob/master/apine.schema.json/participant", 
+      "type": "string", 
+      "title": "participant", 
+      "description": "The corresponding participant"
```

Comment:
if string, add `minlength=1`. How about making it an integer, since this is what BIDS uses?

Reply:
@glatard BIDS supports strings here, too. Most of the `ds*` examples just happen to use integers.